index.js

```diff
@@ -25,7 +27,7 @@
      * on opacity:0 or visibility:hidden.
      */
     checkComponentVisibility: function() {
-      var domnode = this.getDOMNode(),
+      var domnode = React.findDOMNode(this),
```

Comment:
Let's pass this in, instead, since the only places we call `checkComponentVisibility`, we already have that domnode available:
```
checkComponentVisibility: function(domnode) {
   ...
}
```